src/hid_freebsd.c

```diff
@@ -4,26 +4,13 @@
  * license that can be found in the LICENSE file.
  */
 
-#include <sys/types.h>
+#include <sys/param.h>
 
 #include <dev/usb/usb_ioctl.h>
 #include <dev/usb/usbhid.h>
-
-#include <sys/param.h>
-
-/*
```

Comment:
My main concern is that those wanting to test might not understand how to.

If you want to keep it that short, I would make it

```
/* see usbhid(4) and hidraw(4) on FreeBSD >= 13 */
```
(as `man 4 usbhid` contains information on how enable usbhid(4), which is required to allow hidraw(4) to attach instead of uhid(4)). Not sure, but I think "on" is better than "in" (but I'm not a native speaker).